example-node-list.txt

```diff
@@ -1,3 +1,3 @@
-node1.example.com:9200
-node2.example.com:9200
-node3.example.com:9200
\ No newline at end of file
+node1.example.com:9200,cluster1,elasticsearch,elasticsearch_cluster1_dev
+node2.example.com:9200,cluster1,elasticsearch,elasticsearch_cluster1_dev
+node3.example.com:9200,cluster1,elasticsearch,elasticsearch_cluster1_dev
```

Comment:
Maybe leave `example-node-list.txt` untouched and add a `example-node-list-expanded.txt`.